examples/null_example/src/null_example.ts

```diff
@@ -0,0 +1,63 @@
+import { int, Query, Update, Variant } from 'azle';
+
+type partially_null_object = {
+    first_item: int;
+    second_item: null;
+    third_item: int;
+};
+
+type two_null_object = {
+    first_item: null;
+    second_item: null;
+};
+
+type three_null_object = {
+    first_item: null;
+    second_item: null;
+    third_item: null;
+};
+
+export function null_function(param: null): Query<null> {
+    return param;
+}
+
+export function get_partial_null_record(): Query<partially_null_object> {
```

Comment:
If we're calling this `get_partial_null_record` then I think we should name the type `PartiallyNullRecord`

Reply:
Sounds good, and I'll change the function to ```get_partially_null_record``` while I'm at it